packages/utils/src/sqs/peek-dlq-print-details.ts

```diff
@@ -0,0 +1,186 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+// keep that ^ on top
+import { BadRequestError } from "@metriport/shared";
+import { SQS } from "aws-sdk";
+import fs from "fs";
+import { Config } from "../../../api/src/shared/config";
+
+/**
+ * This script saves the details of the messages in the DLQ to a file.
+ */
+
+// Link to the FHIR Server DLQ (https://sqs....)
+const fhirServerDlqUrl = "";
```

Comment:
Maybe use `.env` so we can have diff versions of the file local and easily switch env w/o having to edit the file?